### PR TITLE
style(qcommon): addresses SpotBugs and some CheckStyle issues

### PR DESCRIPTION
**Pull request by holmeso**
# Description

Clean-up of `qcommon` focussing on SpotBugs issues and some CheckStyle ones too.
Removed some used code and tests.

## Type of change

Please delete options that are not relevant.

- [X] SpotBugs and CheckStyle fixes 

# How Has This Been Tested?

Existing tests pass

# Are WDL Updates Required?

No

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: qcommon/test/org/qcmg/common/util/SnpUtilsTest.java [hunk 1]
```diff
@@ -108,24 +108,6 @@ public void getBaseCountsACs() {
 		
 	}
 	
-	@Test
```
Comment (ChristinaXu2017) [inline qcommon/test/org/qcmg/common/util/SnpUtilsTest.java:111]: I suggest changing SnpUtils.getCompoundSnpDistribution(String)  to SnpUtils.getCompoundSnpDistribution(String , 0). 
Even the method of getCompoundSnpDistribution(String) is never used, but getCompoundSnpDistribution(String, int) is used. It is good to test when the second value is 0.

File: qcommon/test/org/qcmg/common/vcf/VcfUtilsTest.java [hunk 2]
```diff
@@ -52,45 +52,6 @@ public class VcfUtilsTest {
 	}
 	
 	
-	@Test
-	public void getAltFrequencyTest() throws Exception{
```
Comment (ChristinaXu2017) [inline qcommon/test/org/qcmg/common/vcf/VcfUtilsTest.java:56]: why delete getAltFrequencyTest()?  you can move it into VcfFormatFieldRecordTest if you think it is not related to VcfUtils.

**Review by ChristinaXu2017 — COMMENTED** [review]
some unit test codes can move to other classes

**Comment by holmeso** [comment]
Thanks for the review, I have pushed another commit reinstating one of the deleted unit tests.

**Review by ChristinaXu2017 — APPROVED** [review]
Looks Good

**Pull request merged**